components/standings/commons/starcraft_starcraft2/standings_storage_starcraft_legacy.lua

```diff
@@ -25,13 +26,17 @@ local INVALID_OPPONENT_CATEGORY = '[[Category:Pages with invalid opponent '
 local Wrapper = {}
 
 function Wrapper.table(frame)
+	if not Wrapper._shouldStore(frame.args) then
```

Comment:
getArgs not needed for table entrypoint?

Reply:
since it is directly invoked in the template it works without
but adding iot doesn't hurt either so i added it